llvm/test/tools/yaml2obj/XCOFF/dwarf.yaml

```diff
@@ -0,0 +1,108 @@
+## Test that we can generate and read DWARF section headers
```

Comment:
Nit: full stops at ends of coments please. Applies throughout.

I suggest renaming this test to dwarf-subsection-types.yaml, because there is scope for other DWARF testing in the future (the ObjectYAML library supports generating DWARF content from YAML, so if you choose to add this to the XCOFF path, you'll want to test it in other tests.